org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/concurrent/RequestManager.xtend

```diff
@@ -101,6 +106,7 @@ class ReadRequest<V> extends AbstractRequest<V> {
 
 @FinalFieldsConstructor
 class WriteRequest<U, V> extends AbstractRequest<V> {
+	static final Logger LOG = Logger.getLogger(WriteRequest);
```

Comment:
why do we use final and not val here?

Reply:
We have prepared the Java port of server.concurrent so this won't be an issue anymore.